packages/flutter_tools/test/commands.shard/hermetic/devices_test.dart

```diff
@@ -23,9 +24,19 @@ void main() {
       Cache.disableLocking();
     });
 
+    MockCache cache;
+
+    setUp(() {
+      cache = MockCache();
+      when(cache.dyLdLibEntry).thenReturn(const MapEntry<String, String>('foo', 'bar'));
```

Comment:
Device discovery created by instantiating DeviceDiscoverers looks up this entry from the cache, will fail if the path does not exist.